src/Cli/dotnet/commands/dotnet-tool/install/ToolInstallGlobalOrToolPathCommand.cs

```diff
@@ -106,6 +115,27 @@ public static T GetValueOrDefault<T>(CliOption<T> option, T defaultOption, Parse
         }
 
         public override int Execute()
+        {
+            if (_all)
+            {
+                var toolListCommand = new ToolListGlobalOrToolPathCommand(
+                    _parseResult
+                    , toolPath => { return _store; }
```

Comment:
Do we need a second argument here?

Reply:
Yes. This is necessary for part of the file system mock